src/components/member/tabs/ClaimsTab.tsx

```diff
@@ -1,202 +1,154 @@
-  handleOpen: () => void
-  handleClaimSubmit: (mutation: any) => void
-  typeChangeHandler: (event: any) => void
-  dateChangeHandler?: (type: string, e: any, value: any) => void
-}
+  if (claims.length === 0) {
+    return (
+      <StandaloneMessage paddingTop="10vh">
+        Claims list is empty
+      </StandaloneMessage>
+    )
+  }
 
-const ClaimsTab: React.FC<ClaimsTabProps> = (props) => {
   return (
     <FadeIn>
-      <Container<State, ActionMap<State, Actions>>
-        initialState={{
-          open: false,
-          date: new Date(),
-          value: 'EMAIL',
-        }}
-        actions={{
-          handleClose: () => (_) => ({ open: false }),
-          handleOpen: () => (_) => ({ open: true }),
-          handleClaimSubmit: (mutation) => (state) => {
-            mutation({
-              variables: {
-                memberId: props.memberId,
-                date: format(state.date, "yyyy-MM-dd'T'HH:mm:ss"),
-                source: state.value,
-              },
-            }).then((response) => {
-              history.push(
-                `/claims/${response.data.createClaim}/members/${props.memberId}`,
-              )
-            })
-            return { date: new Date(), value: 'EMAIL', open: false }
-          },
-          typeChangeHandler: (event) => (_) => ({
-            value: event.target.value,
-          }),
-          dateChangeHandler: (date) => {
-            return {
-              date,
-            }
-          },
-        }}
-      >
-        {({
-          handleClose,
-          handleOpen,
-          dateChangeHandler,
-          typeChangeHandler,
-          handleClaimSubmit,
-          open,
-          value,
-          date,
-        }) => (
-          <>
+      <HeaderWrapper>
+        <MainHeadline>
+          Claims
+          <RefreshButton onClick={() => refetch()} loading={loading}>
+            <ArrowRepeat />
+          </RefreshButton>
+        </MainHeadline>
+
+        <FormWrapper>
+          {showForm ? (
+            <>
+              <EnumDropdown
+                enumToSelectFrom={ClaimSource}
+                placeholder={'Claim Source'}
+                setValue={(source) => setClaimSource(source)}
+              />
+              <Spacing left={'small'} />
+              <DateTimePicker
+                disabled={createClaimLoading}
+                date={claimDate}
+                fullWidth={true}
+                setDate={(date) => setClaimDate(date)}
+                placeholder="Notification date"
+                maxDate={new Date()}
+              />
+              <Spacing left={'small'} />
+              <Button
+                disabled={createClaimLoading}
+                variation="danger"
+                color="danger"
+                onClick={() => {
+                  setShowForm(false)
+                  setClaimDate(null)
+                  setClaimSource(null)
+                }}
+              >
+                Cancel
+              </Button>
```

Comment:
I would wrap the button with the spacing here.
```suggestion
              <Spacing left={'small'}>
              <Button
                disabled={createClaimLoading}
                variation="danger"
                color="danger"
                onClick={() => {
                  setShowForm(false)
                  setClaimDate(null)
                  setClaimSource(null)
                }}
              >
                Cancel
              </Button>
              </Spacing>
```